pkg/kubelet/dockertools/container_labels.go

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I think the"labels.go" is clear enough that we don't need to add the "container_" prefix :)